nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -25,6 +25,50 @@ class MLTuning(ExperimentFunction):
     """Class for generating ML hyperparameter tuning problems.
     """
 
+    def get_dataset(self, dimension):
+
+        num_data: int = 120  # Training set size.
+        self.num_data = num_data
+        
+        # Training set.
+        X = np.arange(0., 1., 1. / (num_data * dimension))
+        X = X.reshape(-1, dimension)
+        random_state = np.random.RandomState(17)
```

Comment:
I think this is still a problem for having robust statistics, using the xp random generator is much better

Reply:
Maybe after the other PR, so that there is  no conflict ? the idea is to use seed from the loop in experiments.py, right ?